drivers/usb/gadget/android.c

```diff
@@ -2437,15 +2602,37 @@ functions_store(struct device *pdev, struct device_attribute *attr,
 		}
 
 		while (conf_str) {
-			name = strsep(&conf_str, ",");
```

Comment:
I believe this should not be removed. If not removed adapted version of the patch works also on iyokan.